cli/src/config/templates.toml

```diff
@@ -39,6 +39,20 @@ log = 'builtin_log_compact'
 op_log = 'builtin_op_log_compact'
 show = 'builtin_log_detailed'
 
+op_log_node = 'if(current_operation, "@", "◉")'
+
+log_node = '''
+label("node",
```

Comment:
nit: perhaps, "node" should be moved to the caller (as we do for "op_log"), but that can be addressed later.